src/api/json/catalog.json

```
@@ -2179,6 +2179,12 @@
       "fileMatch": ["**/assets/*/sounds.json"],
       "url": "https://raw.githubusercontent.com/AxoCode/json-schema/master/minecraft/sounds.json"
     },
+    {
+      "name": "MkDocs Configuration",
+      "description": "MkDocs Configuration Schema, see [mkdocs schema](https://42picky.com/other/schemas/)",
+      "url": "https://42picky.com/assets/f/schemas/mkdocs.schema.json",
+      "fileMatch": ["mkdocs.yml"]
```

```suggestion
      "fileMatch": ["mkdocs.yml", "mkdocs.yaml"]
```

Optional fix: if both extensions are valid, add them both.